ruby_executable/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Even being familiar with the approach you're taking here, when I was trying to navigate to one of the scripts, out of habit I looked here first and was confused to find an empty file. A comment stating that code lives in the `bin` folder would have made me realize my mistake a bit quicker 😅